.circleci/config.yml

```diff
@@ -56,6 +53,9 @@ jobs:
                   paths:
                       - dist
                       - components/coverage
+            - run:
```

Comment:
I'm assuming we're leaving the `cat` commands in here for testing memory usage over time right?

Reply:
yes just a temp item for now